simple_config

```diff
@@ -2,6 +2,8 @@
 
 port 2020;  # This is also a comment.
 
+numThreads 5;
```

Comment:
Yes, we had agreed on `snake_case` earlier, I think I just had a mental lapse when writing this.